library/std/src/io/error.rs

```diff
@@ -400,6 +400,11 @@ pub enum ErrorKind {
     #[stable(feature = "out_of_memory_error", since = "1.54.0")]
     OutOfMemory,
 
+    /// The operation was partially successful and needs to be checked
+    /// later on due to not blocking
```

Comment:
```suggestion
    /// later on due to not blocking.
```
while you're already at it, sorry